src/App.js

```diff
@@ -37,10 +43,55 @@ class App extends React.Component {
     })
   }
 
+  updateInputLanguage = (input) => {
+    this.setState({
+        inputLanguage: input.target.value
+    });
+  }
+
+  updateOutputLanguage = (input) => {
+    this.setState({
+        outputLangauge: input.target.value
+    });
+  }
+
   render() {
     return (
       <div>
         <Navbar/>
+        
+       <div class="selectlanguage">
+       <FormControl>
+        <InputLabel id="demo-controlled-open-select-label">Input</InputLabel>
+       <Select
+          value={this.inputLanguage}
+          onChange={this.updateInputLanguage}
+        >
+          <MenuItem value="">
+            <em>None</em>
+          </MenuItem>
+          <MenuItem value={"js"}>JavaScript</MenuItem>
+          <MenuItem value={"py"}>Python</MenuItem>
+        </Select>
+        </FormControl>
+        
+       
+        <FormControl>
+        <InputLabel id="demo-controlled-open-select-label">Output</InputLabel>
```

Comment:
Is this id being used for something? It seems to be duplicated too.

Reply:
I pasted from a material ui example - I'll remove it